### DISCUSSION TITLE
Is is possible use MONAI Label for Nasal segmentation?

### DISCUSSION THREAD

**Discussion by paper-rest-ssl** [Project-MONAI/tutorials#1447] [Q&A] [answered]
Hi!
Is it appropriate to use MONAI Label with 3D Slicer for training the model of the **nasal airway cavity** segmentation? I saw that there are some pre-trained models for "whole body", "lung", "multiorgan", "pancreas and tumour". However, I couldn't find any specifically for the nasal airway system. My second question is, if there are no pre-trained models, is it possible for me to train the model myself using MONAI Label?

Best Regards

**Comment by KumoLiu** [comment]
Hi @Ruslan0110, I think you can train any dataset by using MONAI.
You can refer to this folder, https://github.com/Project-MONAI/tutorials/tree/main/3d_segmentation.
Also for the integration with MONAILabel, you can refer this folder, https://github.com/Project-MONAI/tutorials/tree/main/monailabel.
Hope it can help you, thanks!

**Reply by paper-rest-ssl** [reply]
Thank you!

**Comment by diazandr3s** [comment, marked as answer]
HI @Ruslan0110,

Thanks for opening this discussion. 
In addition to what @KumoLiu suggested, you could also train a model directly in MONAI Label by following these instructions: https://www.youtube.com/watch?v=3HTh2dqZqew

Using the [Radiology](https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology) app and the [Segmentation](https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#segmentation) model. You could follow these steps:

- Modify/update the label names here: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L37-L63
- As there is no pretrained model for Nasal segmentation, set this flag to False: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L72
- Organize the images and labels in MONAI Label structure following these instructions https://github.com/Project-MONAI/MONAILabel#step-4-data-preparation
- Start the MONAI Label server and trigger the training.

You can find more information on this video (https://youtu.be/-HAryYAO5J4?t=331) or on the Project MONAI YouTube channel or in the documentation.

Hope this helps,

**Reply by paper-rest-ssl** [reply]
Thank you!
